bridge/src/main/java/io/stargate/bridge/service/ExceptionHandler.java

```diff
@@ -160,6 +189,7 @@ private void handleUnavailable(UnavailableException ue) {
   }
 
   private void handleWriteTimeout(WriteTimeoutException wte) {
+    LOG.error(getErrorMessage(Status.DEADLINE_EXCEEDED, wte));
```

Comment:
timeouts are not the same as in the public or? Can you adapt so it s same 1:1?

Reply:
timeouts are handled in the same way as in public, both public and bridge follow https://github.com/stargate/stargate/pull/2225#issuecomment-1311294546 impl matrix for logging and stacktrace

Reply:
my bad